spec/system/user/user_edits_integrations_spec.rb

```diff
@@ -33,5 +33,20 @@
       visit "/settings/integrations"
       expect(find_field("Twitch Username").value).to eq "TestTwitchUser"
     end
+    it "has connect-to-stackbit prompt" do
+      visit "/settings"
+      expect(page).to have_current_path("/settings")
+
+      click_link "Integrations"
+      expect(page).to have_text("Connect to Stackbit")
+    end
+    xit "has connected-to-stackbit prompt if already integrated" do
+      create(:doorkeeper_access_token, resource_owner: user)
```

Comment:
I suppose this test is failing because you check for `access_grants` but only create an `access_token`.
Maybe we should check for access_tokens?

Reply:
As I understood, `access_grant` only lives for a short time
https://stackoverflow.com/questions/28176698/difference-between-access-grant-and-access-token#answer-28981818

Reply:
Good call. Thanks for the clarification.